style.css

```diff
@@ -398,3 +444,172 @@ textarea:focus {
 .contact-me-form-button {
   align-self: flex-start;
 }
+
+/* Desktop version of site */
+@media screen and (min-width: 768px) {
+  .logo > a {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #6070ff;
+    margin: 0 0 0 100px;
+    font-size: 18px;
+  }
+
+  .headline-main {
+    border-radius: 0 0 0 4em;
+    background-color: #fff;
+    background-image: url(./assets/Header\ bg.png);
+    background-size: 96%;
+    background-position-x: center;
+    background-repeat: no-repeat;
+    height: 100vh;
+  }
+
+  .headline {
+    margin: 10px 0;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 12px;
+    width: 751px;
+    height: 268px;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .nav-links {
+    position: absolute;
+    top: 8px;
+    right: 120px;
+    display: flex;
+    margin-top: 15px;
+    padding: 0;
+    color: #6070ff;
+    font-size: 18px;
+  }
+
+  .nav-link {
+    margin: 0 10px;
+    cursor: pointer;
+    color: #344563;
+    font-size: 15px;
+  }
+
+  .social-icons {
+    margin-block: 10px;
+  }
+
+  header {
+    padding: 23px;
+    padding-bottom: 75px;
+  }
+
+  .headline-heading {
+    align-items: center;
+    color: #172b4d;
+  }
+
+  .works-card > img {
+    width: 100%;
+    height: 400px;
+  }
+
+  .works-card:nth-child(even) > img {
+    order: 2;
+  }
+
+  .works-card:nth-child(even) > .works-card-main {
+    order: 1;
+  }
+
+  .works-card {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 2.5em;
+    padding: 1em;
+  }
+
+  .headline-skill-container {
+    display: flex;
+    flex-direction: row;
+    height: 590px;
+  }
```

Comment:
- Kindly, the About section left takes 40% and the right one the remaining 60% of the page's width, and the contents of each of them are aligned to the right, so please fix it to stick to the Figma design and have better UI 👍
- Please minimize the tech card size to stick the figma design 👍 
### Your Design 👇
<img width="917" alt="Screenshot 2023-06-14 195923" src="https://github.com/Afsheen96/Portfolio/assets/52010789/a2ef2422-d8e8-40d8-b4d2-15dd23661a00">


### Figma Design 👇
<img width="814" alt="Screenshot 2023-06-14 195947" src="https://github.com/Afsheen96/Portfolio/assets/52010789/530c1715-7d6c-48be-b792-71a318a404cb">